_includes/tutorials/joining-stream-stream/ksql/code/tutorial-steps/dev/print-output-topic.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
```

Comment:
Can we switch this up to a `PRINT` statement? The result looks a bit more clunky, but it does help people realize that the resulting stream is just a topic that can be used like any other.